examples/customize/build_graph/components/writers/custom_writer.py

```diff
@@ -6,11 +6,12 @@
 from neo4j_graphrag.experimental.components.kg_writer import KGWriter, KGWriterModel
 from neo4j_graphrag.experimental.components.types import LexicalGraphConfig, Neo4jGraph
 from pydantic import validate_call
+from neo4j_graphrag.utils import telemetry
```

Comment:
I'm not sure this should be classified as telemetry, so this name is a bit misleading. 
I'd say it's more of a driver configuration change. So maybe name the module `driver_config` or similar?